win/travis-build.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Adapted from the Debian travis-build.sh to test building a Windows binary
+# according to the build instructions in README.md
+
+set -xe
+shopt -s extglob
+
+TRAVIS_BUILD_STEP="$1"
+
+if [ "$TRAVIS_BUILD_STEP" == "install" ]; then
+    sudo apt-get update -q
+    sudo apt-get install -y devscripts cdbs osc
+
+    if test "$encrypted_585e03da75ed_key" -a "$encrypted_585e03da75ed_iv"; then
```

Comment:
@ivaradi I removed the PPA/OBS stuff, but I wasn't sure if I should pull out this block

Reply:
Lines 1-11 are needed (6 and 7 are not strictly necessary, but are useful). Lines 12-13 install some packages needed to build and upload the Debian package, so they can also be removed. The block frome line 15 to 23 sets up the keys to sign the Debian source package, so are again unnecessary. Lines 44-59 download the Debian upstream package, if there is one, if not, they create a new one, so they are also useless for the Windows build.